app/code/Vladimirl/Chatter/Block/Data.php

```diff
@@ -0,0 +1,40 @@
+<?php
+declare(strict_types=1);
+
+namespace Vladimirl\Chatter\Block;
+
+use Magento\Framework\View\Element\Template;
+use Magento\Framework\View\Element\Template\Context;
+
+class Data extends Template
```

Comment:
@Vladimir-L Use full class names in `extends`, `implements`, in constructor and in the property definition.